email_template_multi_report/models/email_template.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# © 2016 Savoir-faire Linux
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import base64
+
+from openerp import api, fields, models
+from openerp.tools.safe_eval import safe_eval
+
+
+class EmailTemplate(models.Model):
+
+    _inherit = 'email.template'
+
+    report_line_ids = fields.One2many(
+        'email.template.report.line', 'template_id', string='Other Reports')
+
+    @api.model
+    def generate_email_batch(self, template_id, res_ids, fields=None):
+        results = super(EmailTemplate, self).generate_email_batch(
+            template_id, res_ids, fields=fields)
+
```

Comment:
@pedrobaeza your comment is very subjective.This is my style of programming, maybe there is a few lines wasted here, but the other extreme is much worse.